source/enterprise/tech-guide/backup-eca-guide.md

```diff
@@ -2,7 +2,7 @@
 layout: default
 title: Backup the Client Administration
 github_link: enterprise/tech-guide/backup-eca-guide.md
-indexed: false
+indexed: true
```

Comment:
Wie abgesprochen mit @dnoegel, bitte ECA Artikel _nicht_ in den Suchindex mit aufnehmen.